.github/workflows/format-code.yml

```diff
@@ -13,7 +13,9 @@ jobs:
           submodules: recursive
 
       - name: Install clang-format
-        run: brew install clang-format
+        run: |
+          eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+          brew install clang-format
```

Comment:
just remove the step altogether? clang-format is already available https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#language-and-runtime